ansible/collections/requirements.yml

```diff
@@ -2,4 +2,4 @@ collections:
   - name: k3s.orchestration
     source: git+https://github.com/k3s-io/k3s-ansible.git
     type: git
-    version: 3e0c982a95717b76b90a405d53a68a5a6a95d316
\ No newline at end of file
+    version: master
```

Comment:
We should pin this to a specific commit.

Reply:
Re-pinned to latest 99fa632acb713758c8ee376e2a6cc9d03404914c